services-gateway-client-transport/src/main/java/io/scalecube/services/gateway/transport/GatewayClientSettings.java

```diff
@@ -3,19 +3,22 @@
 import io.scalecube.net.Address;
 import io.scalecube.services.exceptions.DefaultErrorMapper;
 import io.scalecube.services.exceptions.ServiceClientErrorMapper;
+import java.time.Duration;
 import reactor.netty.tcp.SslProvider;
 
 public class GatewayClientSettings {
 
   private static final String DEFAULT_HOST = "localhost";
   private static final String DEFAULT_CONTENT_TYPE = "application/json";
+  private static final Duration DEFAULT_KEEPALIVE_INTERVAL = Duration.ofSeconds(30);
```

Comment:
Do we need this default value?
So, am I right that ping-pong mechanism is enabled in any case?